CIP-XXXX/CIP-XXXX.md

```diff
@@ -0,0 +1,916 @@
+---
+CIP: ?
+Title: Cardano dApp-Wallet Web Bridge Governance Extension
+Category: Wallets
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/509
+    - https://discord.com/channels/826816523368005654/1101547251903504474/1101548279277309983
+Created: 2022-02-24
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This document describes an interface between webpage/web-based stacks and
+Cardano wallets. This specification defines the API of the javascript object
+that needs to be injected into web applications.
+
+These definitions extend
+[CIP-30 | Cardano dApp-Wallet Web Bridge](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0030)
+to provide support for
+[CIP-1694? | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/pull/380)
+focussed web-based stacks. Here we aim to support the requirements of both Ada
+holders' and DReps' interactions with such web-based stacks.
+
+> **Note** This proposal assumes knowledge of the ledger governance model as outlined within
+> [CIP-1694?](https://github.com/cardano-foundation/CIPs/pull/380).
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 introduces many new concepts, entities and actors to Cardano;
+describing their implementation at the ledger level. Yet, for most eco-system
```

Comment:
```suggestion
describing their implementation at the ledger level. Yet, for most ecosystem
```
I wouldn't disqualify your using the hyphen but this is an irregular spelling of _ecosystem_ & the first time I've seen it in Cardano reference material:
- https://www.merriam-webster.com/dictionary/ecosystem
- https://en.wikipedia.org/wiki/Ecosystem
- https://cardano-foundation.github.io/state-of-the-developer-ecosystem/2022/
- https://landing.essentialcardano.io/guide-to-the-ecosystem
- https://developers.cardano.org/showcase/?tags=ecosystem

Reply:
I agree, good spot -- not sure where I got that from 🤠